backend/src/middleware/userMiddleware.js

```diff
@@ -11,6 +13,10 @@ export default {
       return result
     },
     UpdateUser: async (resolve, root, args, context, info) => {
+      const USERNAME_MIN_LENGTH = 3 // TODO move to the correct place
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-02T17:46:40Z" title="Thursday, May 2nd 2019, 7:46:40 pm +02:00">May 2, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -11,6 +13,10 @@ export default {
       return result
     },
     UpdateUser: async (resolve, root, args, context, info) => {
+      const USERNAME_MIN_LENGTH = 3 // TODO move to the correct place
```


When I see [magic numbers](https://en.wikipedia.org/wiki/Magic_number_%28programming%29) I always wonder what's the justification for `3` :thinking: how about: "Username can't be blank" ? Blank === only whitespace.

@ulfgebhardt is it still possible to have a username with three blanks?
```
username:  "   "
```

Reply:
> [<img alt="ulfgebhardt" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-05-06T10:59:11Z" title="Monday, May 6th 2019, 12:59:11 pm +02:00">May 6, 2019</time>_

---


Yes that is possible I guess, we do not have a whitespace trim or anything alike